src/services/stepsService.ts

```diff
@@ -286,6 +286,15 @@ export class StepsService {
     return flattenMembers.concat(children.length ? StepsService.flattenSteps(children) : children);
   }
 
+  getFollowingStep(stepUuid?: string) {
+    if (!stepUuid) return undefined;
+    const currentStepIdx = this.findStepIdxWithUUID(stepUuid);
+    const steps = useIntegrationJsonStore.getState().integrationJson.steps;
+    return currentStepIdx !== -1 && steps.length > currentStepIdx + 1
```

Comment:
[side-comment] At the beginning I thought that we needed `steps.length >= currentStepIdx + 1` but after calculation, I think that it's ok 😄 

I went through this thought process:

Having this array: `[ 'a', 'b' ]`
**Case A:** Using `index=0` meaning `'a'`, for the condition `steps.length > currentStepIdx + 1` this would be `2 > 0 + 1` so yeah, 2 > 1.

**Case B:** Taking `'b'`, then it will be `2 > 1 + 1`, which is not `true` and it's ok since there's no additional step 🙆‍♂️